.ci_support/linux_.yaml

```diff
@@ -0,0 +1,53 @@
+c_compiler:
+- toolchain_c
+cxx_compiler:
+- toolchain_cxx
+dbus:
+- '1'
+fontconfig:
+- '2.13'
+freetype:
+- 2.8.1
+gst_plugins_base:
+- '1.12'
+icu:
+- '58'
+jpeg:
+- '9'
+libpng:
+- 1.6.34
+openssl:
+- 1.0.2
+perl:
+- '5.26'
+pin_run_as_build:
+  dbus:
+    max_pin: x
+  fontconfig:
+    max_pin: x
+  freetype:
+    max_pin: x.x
+  gst-plugins-base:
+    max_pin: x.x
+  icu:
+    max_pin: x
+  jpeg:
+    max_pin: x
+  libpng:
+    max_pin: x.x
+  openssl:
+    max_pin: x.x.x
+  perl:
+    max_pin: x.x.x
+  sqlite:
+    max_pin: x
+  xz:
+    max_pin: x.x
+  zlib:
+    max_pin: x.x
+sqlite:
+- '3'
+xz:
+- '5.2'
```

Comment:
This is actually a bogus pinning b/c `xz` is only used to extract `qtwebkit`, but this is fine for now.